src/test/system_tests/test_utils.js

```diff
@@ -73,4 +73,62 @@ function blocks_exist_on_cloud(need_to_exist, pool_id, bucket_name, blocks, s3)
         });
 }
 
+async function create_hosts_pool(
```

Comment:
Change filename to test_utils_agents or something similar to indicate handling of pools/agents?

Reply:
It is a general file for utils to be used in tests, you can see that blocks_exist_on_cloud was already there before the change. I don't see any need to separate at the current moment (only 3 utils are implemented)

Reply:
General utils for tests should be under /test/utils and should be separated according to areas of concern